rules/falco_rules.yaml

```diff
@@ -2114,7 +2114,7 @@
   condition: >
     spawned_process and container and
     ((proc.name = "nc" and (proc.args contains "-e" or proc.args contains "-c")) or
-     (proc.name = "ncat" and (proc.args contains "--sh-exec" or proc.args contains "--exec"))
+     (proc.name = "ncat" and (proc.args contains "--sh-exec" or proc.args contains "--exec" or proc.args contains "-e" or proc.args contains "-c"))
```

Comment:
How about "-e " and "-c " to make sure you don't match args that start with -c, like -chat?

Reply:
since we will evaluate the `proc.name` first,  the existing evaluation for arguments will be evaluated later, then it should be fine. 

I have a minor request: can you please also add `--lua-exec` in the argument evaluation?

Reply:
I believe @mstemm's suggestion is valid because ncat has a `--chat` option, which gets caught by my current proposed fix. I'll fix it.

Yes @Kaizhe I'll add the `--lua-exec` option also, good catch!